src/openforms/utils/pdf.py

```diff
@@ -106,5 +106,5 @@ def render_to_pdf(template_name: str, context: dict) -> tuple[str, bytes]:
         url_fetcher=UrlFetcher(),
         base_url=settings.BASE_URL,
     )
-    pdf: bytes = html_object.write_pdf()
+    pdf: bytes = html_object.write_pdf(pdf_variant="pdf/ua-1")
```

Comment:
can you contribute this patch back to our default-project template?

Reply:
Yes, i've added a PR to the default-project project 😄